src/pragha-playlist.c

```diff
@@ -2246,6 +2247,49 @@ pragha_playlist_insert_mobj_list(PraghaPlaylist *cplaylist,
 	g_signal_emit (cplaylist, signals[PLAYLIST_CHANGED], 0);
 }
 
+/* Append a list of mobj to the current playlist and play the first*/
+void
+pragha_playlist_append_mobj_list_and_play(PraghaPlaylist *cplaylist, GList *list)
+{
+	PraghaMusicobject *mobj;
+	gint prev_tracks = 0;
+	GtkSortType order;
+	gint column;
+	GList *l;
+
+	prev_tracks = pragha_playlist_get_no_tracks(cplaylist);
+
+	/* TODO: pragha_playlist_set_changing() should be set cursor automatically. */
+	set_watch_cursor (GTK_WIDGET(cplaylist));
+	pragha_playlist_set_changing(cplaylist, TRUE);
+	gtk_tree_view_set_model(GTK_TREE_VIEW(cplaylist->view), NULL);
+
+	gtk_tree_view_set_model(GTK_TREE_VIEW(cplaylist->view), cplaylist->model);
+
+	gint i=0;
+	for (l = list; l != NULL; l = l->next) {
+		mobj = l->data;
+		if(i==0){
+			pragha_playlist_append_mobj_and_play(cplaylist, mobj);
+			
```

Comment:
About code:
 * Should save the reference to the first mobj, add the entire list, and play the first mobj.

About styles:
 * Space between assignments and comparisons..  e.g: 'if(i==0){' -> 'if (i == 0){'  and 'gint i=0;'   ->  'gint i = 0;'
 * Do not leave unnecessary blank lines ..
 * spaces between brackets and conditionals..  e.g: '}else{'     ->     '} else {'

Reply:
Thanks for your reply . 
Check  this commit [83b636f ](https://github.com/pragha-music-player/pragha/pull/131/commits/83b636f01cd8da8f6c193aa736c4aa508e2b60dd) please.